internal/storage/hub/bundle.go

```diff
@@ -250,6 +239,31 @@ func (b *Bundle) GetAllMatching(ctx context.Context, modIDs []namer.ModuleID) ([
 	return res, nil
 }
 
+func (b *Bundle) getMatch(id namer.ModuleID, withFileStat bool) (*runtimev1.RunnablePolicySet, error) {
```

Comment:
Sorry for causing confusion. It's probably not worth having the `withFileStat` boolean parameter here. They always make things tricky to read and I think the `stat` call has negligible overhead in this context because Go reads the zip file manifest into memory anyway.